NER_Data_collection_and_annotation.ipynb

```diff
@@ -0,0 +1,981 @@
+{
```

Comment:
<p>Nice touch using the progress bar!</p><p> </p><p>Also nitpick: typically docstrings will define variable type for further clarity.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meetnaren/ner-project/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Thanks! Will update the docstring.</p>